content/docs/replication/deployment/powerstore.md

```diff
@@ -172,17 +174,49 @@ parameters:
 
 After creating storage class YAML files, they must be applied to your Kubernetes clusters with `kubectl`.
 
+Creating storage classes for file-type storage is very similar to the process for creating block-type storage classes. You can find sample replication enabled storage class for file storage (File Systems) in the driver repository
+[here](https://github.com/dell/csi-powerstore/blob/main/samples/storageclass/powerstore-file-replication.yaml).
+
+It will look like this:
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "powerstore-replication"
+provisioner: "csi-powerstore.dellemc.com"
+reclaimPolicy: Retain
+volumeBindingMode: Immediate
+parameters:
+  csi.storage.k8s.io/fstype: "ext4"
+  replication.storage.dell.com/nasName: "nas-server"
+  replication.storage.dell.com/allowRoot: "false"
+  replication.storage.dell.com/isReplicationEnabled: "true"
+  replication.storage.dell.com/remoteStorageClassName: "powerstore-replication"
+  replication.storage.dell.com/remoteClusterID: "tgt-cluster-id"
+  replication.storage.dell.com/remoteSystem: "RT-0000"
+  replication.storage.dell.com/rpo: Five_Minutes
+  replication.storage.dell.com/ignoreNamespaces: "false"
+  replication.storage.dell.com/volumeGroupPrefix: "csi"
+  arrayID: "Unique"
+```
+
+Most of these parameters are the same as the above block-storage example, but there are some new ones. Let's go over each file-storage only parameter:
+* `replication.storage.dell.com/nasName` selects the NAS server that the storage class should place created filesystems on. It **must** already exist on the storage array, and it **must** be configured for array-to-array replication.
+* `replication.storage.dell.com/allowRoot` if set to `true` will allow root users to exercise root privileges on the NFS server. Defaults to false.
+
+Otherwise, generating storage class pairs and applying them using `kubectl` is done in the same way as block storage. The nasName value and allowRoot should be consistent from source to target storage classes.
+
 #### Storage Class Creation With repctl
 
 `repctl` can simplify storage class creation by creating a pair of mirrored storage classes in both clusters
-(using a single storage class configuration) in one command.
+(using a single storage class configuration file) in one command.
 
 To create storage classes with `repctl` you need to fill the config with necessary information.
-You can find an example in [here](https://github.com/dell/csm-replication/blob/main/repctl/examples/powerstore_example_values.yaml), copy it, and modify it to your needs.
+You can find an example for block storage [here](https://github.com/dell/csm-replication/blob/main/repctl/examples/powerstore_block_example_values.yaml) and for file storage [here](https://github.com/dell/csm-replication/blob/main/repctl/examples/powerstore_file_example_values.yaml), copy it, and modify it to your needs.
```

Comment:
can you please check whether link is working?

Reply:
Same as above. Link is to a file in a PR that will not be merged in until these changes are pulled in as well, which will be when the feature is complete.